docs/implementations.rst

```diff
@@ -65,6 +89,14 @@ original driving force behind the ASGI project. Supports HTTP and WebSockets
 with Django integration, and any protocol with ASGI-native code.
 
 
+Esmerald
+---------------
```

Comment:
This underline is too long - should match the title above it

Reply:
Oh, totally missed that. Thank you @andrewgodwin 👍🏼